web/src/App.tsx

```diff
@@ -1,141 +1,10 @@
-import React, { useEffect, useContext, useState } from 'react'
-import { AuthContext, AuthProvider, TAuthConfig } from 'react-oauth2-code-pkce'
-import TodoAPI from './api/TodoAPI'
-import { AddTodoResponse, GetTodoAllResponse } from './api/generated'
-import styled from 'styled-components'
-
-export const ToDoContainer = styled.div`
-  width: 400px;
-  padding: 30px;
-`
-
-export const ToDoItem = styled.div`
```

Comment:
Why remove the "demo app"?

Reply:
Some stuff I found and fixed during the new init: removed default react icon from manifest and added equinor icons, disabled rules that would cause conflicts between prettier and eslint, removed unnecessary React imports, made sure eslint wouldn't cause errors on missing React imports (this isn't required after React v17), fixed conflicting rules, removed ignore-files that ignores files that are already default ignored. Removed eds-tokens (for use in projects without react), removed bootsrap etc.

Reply:
Sounds good. If you are happy with the new base, you can add the refactored demo app to this PR